crates/spark/schema/spark.graphql

```diff
@@ -344,6 +355,9 @@ enum CurrencyUnit {
   """Indian Rupee."""
   INR
 
+  """Tether"""
+  USDT
```

Comment:
Should add this to the `CurrencyUnit` enum
https://github.com/breez/spark-sdk/blob/5c3110aaf394340f473d822aaba482580be997df/crates/spark/src/ssp/graphql/models.rs#L117

Reply:
Done